core/src/main/scala/org/ensime/core/Analyzer.scala

```diff
@@ -80,8 +81,8 @@ class Analyzer(
       case Some(scalaLib) => settings.bootclasspath.value = scalaLib.getAbsolutePath
       case None => log.warning("scala-library.jar not present, enabling Odersky mode")
     }
-    settings.classpath.value = config.compileClasspath.mkString(JFile.pathSeparator)
-    settings.processArguments(config.compilerArgs, processAll = false)
+    settings.classpath.value = module.compileClasspath.mkString(JFile.pathSeparator)
```

Comment:
this doesn't include all the transitive dependencies of the module. I think we need some coverage around this if it's currently passing. e.g. what if module A depends on B, and A is using a dependency that is pulled in by B?